caom2pipe/astro_composable.py

```diff
@@ -163,6 +165,37 @@ def check_fits(fqn):
     return True
 
 
+def check_fitsverify(fqn):
+    """
+    Execute fitsverify on fqn
+    :return: bool True if compliant, False if Error count > 0
+    """
+    result = True
```

Comment:
Default is `True`.  Shouldn't it be `False`?

Reply:
Good catch. Thanks.